converters/google/resources/custom_org_policy.go

```diff
@@ -0,0 +1,184 @@
+package google
+
+import (
+	"fmt"
+	"strings"
+)
+
+func resourceConverterCustomOrgPolicy() ResourceConverter {
+	return ResourceConverter{
+		Convert:           GetCustomOrgPolicyCaiObject,
+		MergeCreateUpdate: MergeCustomOrgPolicy,
+	}
+}
+
+func GetCustomOrgPolicyCaiObject(d TerraformResourceData, config *Config) ([]Asset, error) {
+
+	assetNamePattern, assetType, err := getAssetNameAndTypeFromParent(d.Get("parent").(string))
```

Comment:
I am pretty sure this should just use `assetNamePattern = "//cloudresourcemanager.googleapis.com/{{parent}}"` in all cases - this is what's causing the diff due to `placeholder-foobar` in the name, because `assetName` is trying to find `{{project_id}}`, `{{folder_id}}`, or `{{organization_id}}` in the terraform data, and it doesn't exist - so it is replacing that variable with a placeholder.